eng/common/TestResources/deploy-test-resources.yml

```diff
@@ -44,7 +64,7 @@ steps:
 
   - ${{ if parameters.PersistOidcToken }}:
```

Comment:
Are you sure you also don't need to do `${{ if eq(parameters.PersistOidcToken, 'true') }}`?

Reply:
When it's typed as a boolean, you can do the conditional like this without the `eq` function.

Reply:
See https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4210530&view=results as an example run